phpunit.xml.dist

```diff
@@ -20,10 +20,6 @@
 		stopOnSkipped="false"
 		verbose="true"
 	>
-	<php>
-        <ini name="display_errors" value="On" />
-        <ini name="display_startup_errors" value="On" />
-    </php>
```

Comment:
@ovidiul It seems like you're adding this while working, and then it's getting committed accidentally?

Copy `phpunit.xml.dist` to `phpunit.xml` and then add these ini lines to `phpunit.xml`. 

By default, [PHPUnit prefers to use `phpcs.xml`](https://phpunit.de/manual/6.5/en/organizing-tests.html#organizing-tests.xml-configuration), so you can put your local-only modifications in there, and it will be used when running your tests locally.

We already [ignore `phpunit.xml`](https://github.com/Automattic/WPCOM-Legacy-Redirector/blob/47500d5bd46be1d1ab2628caf80ea3c1a638ff39/.gitignore#L6) in the `.gitignore`, so this new file won't get committed.

The same trick can be done with PHPCS config files as well.

Reply:
Thanks Gary for the note, done that already now. The code should be left out of this PR.